stable/jenkins/values.yaml

```diff
@@ -81,3 +82,10 @@ Persistence:
 #  - mountPath: /var/nothing
 #    name: nothing
 #    readOnly: true
+
+NetworkPolicy:
+  # Enable creation of NetworkPolicy resources.
+  Enabled: true
```

Comment:
Is there any harm in setting it to True? Users should notice no difference, whether they have a network plugin that enforces policy or not.